R/app-ui.R

```diff
@@ -12,6 +12,9 @@ app_ui <- function(request) {
 
     dashboardSidebar(
       sidebarMenu(
+        conditionalPanel(!is.na(config::get("path_to_markdown")) == TRUE, {
+            menuItem("Using the App", tabName = "vignette")
+        }),
```

Comment:
@Aryllen Do you know how to add a conditional `menuItem()`? I googled around and tested some `if statements` to no avail. In the case of the code as written, the menu item is not clickable but the text "Using the App" is still displayed.

Reply:
I should add I was testing this with the `config.yml` set to `path_to_markdown: !expr NA`

Reply:
I'm not sure a `conditionalPanel` would work in that case. What you could try is to make a function that checks if the `menuItem` should be created. If so, it gives back the code generated by `menuItem("Using the App", tabName = "vignette")`. If not, it returns nothing.

Reply:
@kelshmo if it's too hard to implement this let's not worry too much about it and just choose one of the vignettes to make the default.

Reply:
Also, this is just a heads up for the future since I often forget myself, but the conditionalPanel needs the condition in javascript ([documentation](https://shiny.rstudio.com/reference/shiny/0.11/conditionalPanel.html)). I think a separate function with an if-else _should_ work, however, and be simpler than conditionalPanels.

Reply:
I swear I tried a good old `if` statement and it was still displaying the text in the browser. Now, it appears to work well...!